jax/scipy/linalg.py

```diff
@@ -264,3 +263,120 @@ def tril(m, k=0):
 @_wraps(scipy.linalg.triu)
 def triu(m, k=0):
   return np.triu(m, k)
+
+@jit
```

Comment:
You can use `@_wraps(scipy.linalg.expm)` to copy the docstring from SciPy -- see also the `lax_description` argument for adding JAX specific details.

Reply:
Okay. I tried using it, but am unable to see `expm` in the docs when I run `sphinx-build -b html -D nbsphinx_execute=never docs docs/build/html`.

Reply:
Okay. I have done that now, but running `sphinx-build -b html -D nbsphinx_execute=never docs docs/build/html` did not result in an entry for `jax.scipy.linalg.expm`.

Reply:
You also have to add an entry to the relevant rst file in our docs.

Reply:
That worked! Thanks for the pointer.